clang/test/Driver/sycl-ftarget-register-alloc-mode.cpp

```diff
@@ -16,6 +16,10 @@
 // RUN:    -fsycl-targets=spir64_gen -ftarget-register-alloc-mode=pvc:default %s 2>&1 \
 // RUN:   | FileCheck -check-prefix=DEFAULT_AOT %s
 
+// RUN: %clang -### -fsycl \
+// RUN:    -fsycl-targets=spir64_gen %s 2>&1 \
+// RUN:   | FileCheck %if system-windows %{ -check-prefix=DEFAULT_AOT %} %else %{ -check-prefix=AUTO_AOT %} %s
```

Comment:
I've never seen the if usage like this - pretty neat.

Reply:
yeah i learned this from @aelovikov-intel i think, it's really useful for cases like this